inc/Module/Build/Sqitch.pm

```diff
@@ -224,4 +225,57 @@ sub fix_shebang_line {
     return $self->SUPER::fix_shebang_line(@_);
 }
 
+sub ACTION_bundle {
+    my ($self, @params) = @_;
+    my $base = $self->install_base or die "No --install_base specified\n";
+    $self->_runtime_cpanfile;
```

Comment:
I don't think this is needed. You can pass args to Carton like `--without develop --without test`.